configuration/pih/scripts/visit/encounterTypeConfig.js

```diff
@@ -1609,7 +1609,7 @@ angular.module("encounterTypeConfig", [])
             editUrl: hfeStandardEditUrl
         };
 
-        // mentalHealth
+        // mentalHealth (mentalHealthFollowup for Liberia)
```

Comment:
line 1615-1622 is the "default" mental health form, the follow up doesn't start until 1625, so this comment seems incorrect?